test/PoseidonGoldenTests/GoldenTestData/forge/ForgePac14/ForgePac14.ind

```diff
@@ -1,3 +1,2 @@
 XXX018	F	POP3
-XXX099	M	POP3
```

Comment:
OK - why is XXX099 not in the output package any more? The selection string is `<XXX099>,<Lamnidis_2018-1.0.1:POP1:XXX017>,<Lamnidis_2018-1.0.0:POP3:XXX018>` and the data in `test/testDat/testPackages/ancient` did not change. I don't understand why it gets excluded.

Reply:
Ha, well that one is quite tricky and points to a potential issue with our current semantics (easy to fix, though). So what's going on here is that `<XXX099>` is not in the _latest_ package, so it slips through the selection. This is the correct behavior given our exact semantics.

Here is the quote:
`<Ind1>: Select all individuals named "Ind1" in all latest packages.`

But I suppose we could change that behaviour to
`<Ind1>: Select the individual(s) named "Ind1" in the latest package where they are available`

Reply:
Actually no, I think our current behavior is correct. If someone takes out an individual in the latest version of a package, then by default we _should_ exclude it... unless the user specifically asks for it using the Specific-ind Syntax.

Also, note that in this case there would be a warning from `determineNonExistentEntites`, so the user would at least be confronted with the fact that this individual cannot be found (which might be surprising, but at least things aren't put under the carpet)

Reply:
OK, to be sure, Just tested this:

```
stephan_schiffels@biomac04 ancient % trident forge -f "<XXX099>" -d . -o ~/Desktop/test
trident v1.3.0.4 for poseidon v2.5.0, v2.6.0, v2.7.0, v2.7.1
https://poseidon-framework.github.io

[Info]    Checking base directories... 
[Info]    Searching POSEIDON.yml files... 
[Info]    5 found
[Info]    Checking Poseidon versions... 
[Info]    Initializing packages... 
[Warning] Duplicate individuals in package collection (XXX011, XXX012, XXX013, ...)
[Info]    Packages loaded: 5
[Info]    Unpackaged genotype data files loaded: 0
[Info]    Forging with the following entity-list: <XXX099>
[Warning] Detected entities that do not exist in the dataset. They will be ignored: <XXX099>
[Info]    0 packages contain data for this forging operation
[Error]   Nothing to be forged
```

OK, so this is a bit confusing as it says `[Warning] Detected entities that do not exist in the dataset. They will be ignored: <XXX099>` which is not true. I will amend the error message.

Reply:
OK, so I've changed that warning. Now it's:

[Warning] The following entities could not be found in the dataset and will be ignored (keep in mind that without further specification, only latest versions of packages are considered): <XXX099>

Reply:
I think I would prefer a tall format for this message, when we're already at it. Happy to add a comment with the following suggestion:

```
[Warning] The following entities could not be found in the dataset and will be ignored
[Warning] <XXX099>
[Warning] <XXX105>
[Warning] Maybe these entities exist in older package versions?
```

What's your opinion?

Reply:
Love it. Will make that change and post here when done.

Reply:
OK, I've made this change now. By the way, why have we not made this an actual error? Shouldn't we fail if the user looks for individuals that cannot be found? That would make sense to me!

Reply:
Yes - probably a good idea to make it an error. I see that you already implemented this now. This means this conversation is resolved, finally :+1: